src/Routes/Devices/UpdateDeviceModal.js

```diff
@@ -120,6 +112,24 @@ const UpdateDeviceModal = ({ updateModal, setUpdateModal, refreshTable }) => {
           {isMultiple ? `${deviceName.length} systems` : deviceName}
         </span>{' '}
         to latest version of the image linked to it.
+        {updateModal.deviceData.some(
+          (device) =>
+            device.deviceStatus !== 'updateAvailable' &&
+            device.deviceStatus !== 'error'
+        ) && (
+          <div className="pf-c-alert pf-m-info pf-m-inline pf-u-mt-md">
```

Comment:
Is there a reason for not using the `Alert` component here?
https://www.patternfly.org/2020.04/documentation/react/components/alert#examples

Reply:
Not any reason that I know of. I can try using that instead!